kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/RecordSerDe.java

```diff
@@ -13,6 +14,9 @@
   class DeserializedKeyValue {
     @Nullable String key;
     @Nullable String value;
+    @Nullable MessageFormat format;
+    @Nullable Integer valueSize;
+    @Nullable String schemaId;
```

Comment:
is it schema id for value or key, I think it's better to pass them separate

Reply:
It's for value. Will add for key also.

Reply:
Done